packages/cli/src/tools/config/index.js

```diff
@@ -124,7 +124,7 @@ function loadConfig(projectRoot: string = process.cwd()): ConfigT {
       get assets() {
         return findAssets(projectRoot, userConfig.assets);
       },
-      platforms: {},
+      platforms: userConfig.platforms,
```

Comment:
This makes sense, why we didn't do it previously?
Btw shouldn't this fall back to empty object?

Reply:
Joi schema automatically makes sure it's empty object by default. So we don't need to.

I believe it was there (we did it for commands). Could've been lost in refactoring.